src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -220,15 +208,16 @@ public static Command rumbleBriefly(Conveyor conveyor) {
    * Assuming a note is here, then runs until it's positioned correctly. Notably, it won't do
    * anything if there's not a note.
    */
-  public static Command finishReceive(Conveyor conveyor) {
+  public static Command finishReceive(Conveyor conveyor, Lights lights) {
     return new SequentialCommandGroup(
             new InstantCommand(
                 () -> conveyor.frontMotor.set(ConveyorCal.RECEIVE_SLOW_SPEED), conveyor),
             new InstantCommand(() -> conveyor.backMotor.set(ConveyorCal.RECEIVE_SLOW_SPEED)),
             new WaitCommand(0.1),
-            new WaitUntilCommand(() -> !conveyor.beamBreakSensorOne.isPressed()),
+            new WaitUntilCommand(() -> conveyor.conveyorBeamBreakSensor.get()),
```

Comment:
True means not broken, false means there's an object

Reply:
Yes so this is what we want

Reply:
It should stop the conveyor once we no longer see a piece